interchain-exchange/tutorial/08-cancel-orders.md

```diff
@@ -38,11 +38,9 @@ func (k msgServer) CancelSellOrder(goCtx context.Context, msg *types.MsgCancelSe
 	}
 
 	// Remove order
-	newBook, err := book.RemoveOrderFromID(msg.OrderID)
-	if err != nil {
+	if err := book.RemoveOrderFromID(msg.OrderID); err != nil {
```

Comment:
```suggestion
	if err := s.Book.RemoveOrderFromID(msg.OrderID); err != nil {
```

@lubtd we may need to update `msg_server_cancelSellOrder.go` and `msg_server_cancelByOrder.go`, because in the updated version we get an order book as `s`.

https://github.com/tendermint/interchange/blob/starport-0.16/x/ibcdex/keeper/msg_server_cancelSellOrder.go#L16

Reply:
Yes actually, sell and buy order books now have a common attribute `OrderBook`

Reply:
fixed, thanks